### ISSUE TITLE
Formalize release process

### ISSUE THREAD

**Issue by dylanmccall** [flathub/org.learningequality.Kolibri#69]
This flatpak is often installed on Endless OS, which includes additional system integration in the host via [eos-kolibri](https://github.com/endlessm/eos-kolibri). As a result, we need to do an additional round of integration testing for any large changes to the flatpak. To achieve this, we have added a [`beta`](https://github.com/flathub/org.learningequality.Kolibri/tree/beta) branch, and test images of Endless OS use `org.learningequality.Kolibri//beta`.

For this to work, it is important that the beta branch is always _equal to_ or _ahead of_ the master branch. We want this testing workflow to be reliable. So, let's establish a release process which guarantees we generate a beta release before pushing to master; that the beta branch isn't missing changes from the master branch, and that major changes are always visible _first_ in EOS test builds.

**Comment by dylanmccall** [comment]
One issue here earlier is our beta branch was a mess because I used it in a cavalier manner and then the flathub folks made everyone's beta branch a protected branch (flathub/flathub#3649).

To unblock this, I did a series of PRs to sync up the master and beta branches - alas, pulling a lot of superfluous commits into master. Now, we can do a workflow like this:

- For _any_ changes, create a pull request with the target branch set to [`beta`](https://github.com/flathub/org.learningequality.Kolibri/tree/beta).
- Once manual testing has completed, or if a developer has decided it is reasonable to skip that step, [create a new pull request](https://github.com/flathub/org.learningequality.Kolibri/compare/master...beta) merging new changes from the `beta` branch into `master`.

I think it's reasonably simple - but we need to make sure we're doing it.
